cmd/nomos/status/client.go

```diff
@@ -354,6 +361,34 @@ func (c *ClusterClient) namespaceRepoClusterStatus(ctx context.Context, cs *Clus
 	}
 }
 
+// IsOssInstallation will check for the existence of ConfigManagement object, Operator deployment, and RootSync CRD
+// If RootSync CRD exist but ConfigManagement and Operator doesn't, it indicates an OSS installation
+func (c *ClusterClient) IsOssInstallation(ctx context.Context, cs *ClusterState) bool {
+	v, cmErr := c.ConfigManagement.Version(ctx)
+	_, operatorDepErr := c.K8sClient.AppsV1().Deployments(configmanagement.ControllerNamespace).Get(ctx, ACMOperatorDeployment, metav1.GetOptions{})
+
+	if cmErr != nil || (operatorDepErr != nil && !apierrors.IsNotFound(operatorDepErr)) {
+		cs.Error = "Failed to check for operator installation status"
+		return false
+	}
+
+	if v != util.NotInstalledMsg && operatorDepErr == nil {
+		return false
+	}
+
+	if v == util.NotInstalledMsg || apierrors.IsNotFound(operatorDepErr) {
+		rootSyncCRDErr := c.Client.Get(ctx, client.ObjectKey{Name: rootSyncCRDName}, &apiextensionsv1.CustomResourceDefinition{})
+		repoSyncCRDErr := c.Client.Get(ctx, client.ObjectKey{Name: repoSyncCRDName}, &apiextensionsv1.CustomResourceDefinition{})
+		if rootSyncCRDErr == nil || repoSyncCRDErr == nil {
+			return true
+		}
+		cs.Error = "Failed to check for Rootsync and Reposync"
```

Comment:
The error message doesn't make sense if `rootSyncCRDErr` or `repoSyncCRDErr` is not_found error.

Reply:
Updated error message to include more details